index.html

```diff
@@ -3817,122 +3817,97 @@ <h3>Introduction</h3>
 					preview).</p>
 			</section>
 
-			<section id="app-toc-html">
+			<section id="app-toc-html" class="informative">
```

Comment:
I think I would move this informative section to _after_ the normative algorithm, but this is largely a matter of taste 😊

Reply:
Ya, I've gone back and forth on this. My thinking is that the number of people reading the appendix for the algorithm is going to be quite small compared to the number of people who will be trying to understand how to structure the table of contents. Over-explaining the algorithm/structure first might bore developers, but it's less of a shock to the average reader.